cfgen/cfgen

```diff
@@ -398,7 +403,7 @@ def enrich_cluster_desc(desc: Dict[str, Any], mock_p: bool) -> None:
         if all(key in node.keys()
                for key in ('processorcount', 'memorysize_mb')):
 
-            logging.debug(
+            LOG.debug(
```

Comment:
I think this is `info`, I don't see `debug` specific information here.

Reply:
Fixed.